release-manifest.yaml

```diff
@@ -32,7 +32,7 @@ ondemand-passenger:
     - ondemand-passenger-dev
     - ondemand-passenger-doc
   versions:
-    - 6.0.23-2.ood{runtime}
+    - 6.0.23-3.ood{runtime}
```

Comment:
If you bump this you'll also need to bump several other places.  I'd recommend just leave this change off and I'll bump in #299 after this is merged.

Reply:
Yes, let's do it that way, I'll push a commit to remove the version bumps soon. Thanks for helping out with this!